.github/ISSUE_TEMPLATE/10-build-installation-issue.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Build/Installation Issue
```

Comment:
Keras does not have its own build process so we can remove this template. Build issues will always be routed to TF.